web/package.json

```diff
@@ -42,21 +41,21 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^11.2.13",
-    "@angular/common": "^11.2.13",
-    "@angular/compiler": "^11.2.13",
-    "@angular/core": "^11.2.13",
-    "@angular/elements": "^11.2.13",
-    "@angular/forms": "^11.2.13",
-    "@angular/platform-browser": "^11.2.13",
-    "@angular/platform-browser-dynamic": "^11.2.13",
-    "@angular/router": "^11.2.13",
-    "@cds/angular": "^5.2.0",
+    "@angular/animations": "^12.1.2",
+    "@angular/common": "^12.1.2",
+    "@angular/compiler": "^12.1.2",
+    "@angular/core": "^12.1.2",
+    "@angular/elements": "^12.1.2",
+    "@angular/forms": "^12.1.2",
+    "@angular/platform-browser": "^12.1.2",
+    "@angular/platform-browser-dynamic": "^12.1.2",
+    "@angular/router": "^12.1.2",
+    "@cds/angular": "^5.4.2",
     "@cds/city": "^1.1.0",
-    "@cds/core": "^5.2.0",
-    "@clr/angular": "^5.2.0",
-    "@clr/icons": "^5.2.0",
-    "@clr/ui": "^5.2.0",
+    "@cds/core": "^5.4.2",
+    "@clr/angular": "^12.0.0-next.0",
```

Comment:
It also looks like 12.0.0 just came out

Reply:
Bumped both Clarity and Angular since new versions are out for both.